.editorconfig

```diff
@@ -37,3 +37,6 @@ max_line_length = unset
 
 [gradle/verification-metadata.xml]
 indent_size = 3
+
+[*.yml]
+ij_yaml_spaces_within_brackets = false
```

Comment:
`yml` files I modified, which got auto-formated by my IDE had unexpected diffs. I'm proposing to add a rule that matches what GitHub does it their docs (i.e. https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) and reformatted all files in a separate commit